### ISSUE TITLE
Filter ONLY current page does not work in SEARCH view

### ISSUE THREAD

**Issue by joel-scaria-hpe-gh** [cars10/elasticvue#275]
**READ BEFORE OPENING AN ISSUE**

> Before you submit an issue please try to reset elasticvue and see if you can reproduce the problem.
> To reset please click `Disconnect and reset` in the footer, this will reset all your saved filters and you have to reconnect to your cluster.
> Proceed and open an issue if your problem persists.

Issue doesnt go away after reset

**Description**

> Please describe your problem here.

"filter only current page" in search view does not work

**Steps To Reproduce**

> List the steps to reproduce your problem:

1. Open elasticvue
2. select search view
3. try to filter for something
4. seeing following error in console [FilterInput.vue:22]
```
TypeError: o[r].toString is not a function
    at filters.ts:25:35
    at Array.some (<anonymous>)
    at filters.ts:23:26
    at c (reactivity.esm-bundler.js:861:19)
    at Array.filter (<anonymous>)
    at ss (reactivity.esm-bundler.js:869:27)
    at Proxy.filter (reactivity.esm-bundler.js:761:12)
    at wo (filters.ts:22:18)
    at v_.fn (SearchResultsTable.ts:93:12)
    at Ym (reactivity.esm-bundler.js:380:28)
```

**Screenshots**

> If applicable, add screenshots to help explain your problem.

![image](https://github.com/user-attachments/assets/8a5e9a32-8652-473b-b1ed-a47b6cad41d4)

**Environment (please include the following information):**
* Elasticsearch version: 7.4.0
* Operating system: Windows 11
* Browser + version: Chrome Version 131.0.6778.86 (Official Build) (64-bit)
* Elasticvue version: 1.1.0 stable
* How are you running elasticvue? (docker, browser extension, web app): extension

**Additional context**

> Anything else?

this was supposed to be fixed in a previous version as part of https://github.com/cars10/elasticvue/issues/244

**Comment by cars10** [comment]
Thanks, will be fixed in `1.1.1`.

**Comment by cars10** [comment]
Fixed in `1.1.1`
